pages/linux/zbarcam.md

```diff
@@ -0,0 +1,20 @@
+# zbarcam
+
+> Scans and decodes barcodes from a video device.
+> More information: <https://linux.die.net/man/1/zbarcam>.
```

Comment:
It's really odd that their website doesn't have CLI docs. By default, we use `manned.org` for man pages.

```suggestion
> More information: <https://manned.org/zbarcam>.
```

Reply:
Alright, good to know. I haven't contributed in a while.